subsys/bluetooth/controller/Kconfig

```diff
@@ -388,7 +388,7 @@ config BT_CTLR_PRIVACY
 	bool "LE Controller-based Privacy"
 	depends on BT_CTLR_PRIVACY_SUPPORT
 	default y
-	select BT_CTLR_FILTER
+	select BT_CTLR_FILTER if (BT_LL_SW_SPLIT || BT_LL_SW_LEGACY)
```

Comment:
`()` aren't obligatory btw (but don't hurt either).

Reply:
I think it is more readable with the `()`, but I may remove it if you don't like it

Reply:
Nah, keep them if it was deliberate.